distribution/lib/Standard/Table/0.0.0-dev/src/Data/Type/Value_Type.enso

```diff
@@ -1,5 +1,6 @@
 from Standard.Base import all
 import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+import Standard.Database.Data.SQL_Type.SQL_Type
```

Comment:
This makes `Standard.Table` library dependent on `Standard.Database`. I'm not sure if we want that because the Database lib already depends on the Table

Reply:
In fact, this caused an issue with IR caching https://github.com/enso-org/enso/actions/runs/4966001323/jobs/8887184345#step:10:3121
What happened is that IR for `Standard.Database.Connection.Database` module was not cached, and it resulted in compilation error:
```
INFO ide_ci::program::command: bashℹ️ In module Standard.Database.Connection.Database:
 INFO ide_ci::program::command: bashℹ️ Compiler encountered errors:
 INFO ide_ci::program::command: bashℹ️ Database.enso[23:11-23:28]: The name `Connection_Details` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[23:33-23:50]: The name `Connection_Options` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[23:55-23:64]: The name `Connection` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[23:68-23:76]: The name `SQL_Error` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[24:25-24:42]: The name `Connection_Options` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[28:29-28:41]: The name `Single_Choice` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[30:28-30:33]: The name `Vector` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[31:63-31:67]: The name `False` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[35:9-35:14]: The name `Option` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[36:5-36:17]: The name `Single_Choice` could not be found.
 INFO ide_ci::program::command: bashℹ️ Database.enso[36:27-36:33]: The name `Display` could not be found.
 INFO ide_ci::program::command: bash⚠️ Aborting due to 11 errors and 0 warnings.
 INFO ide_ci::program::command: bashℹ️ Execution finished with an error: Compilation aborted due to errors.
 ```

Reply:
> This makes Standard.Table library dependent on Standard.Database

Cyclic dependencies are bad. It'd be good to prevent them with some failure - #6679

> In fact, this caused the [issue with IR caching](https://github.com/enso-org/enso/actions/runs/4966001323/jobs/8887184345#step:10:3121)

Oh, my dear! I was hunting the problem for almost a week and it is caused by such a _little oversight_. Thank you very much for working on this PR in parallel with #6584 - it was very refreshing and useful to see different approach to the problem. Without your help I'd still be blindly searching for a bug in `BindingsMap`.